src/main/java/de/tum/in/www1/artemis/service/ModelAssessmentConflictService.java

```diff
@@ -0,0 +1,231 @@
+package de.tum.in.www1.artemis.service;
+
+import java.time.ZonedDateTime;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import de.tum.in.www1.artemis.domain.*;
+import de.tum.in.www1.artemis.domain.enumeration.EscalationState;
+import de.tum.in.www1.artemis.domain.modeling.ConflictingResult;
+import de.tum.in.www1.artemis.domain.modeling.ModelAssessmentConflict;
+import de.tum.in.www1.artemis.repository.ConflictingResultRepository;
+import de.tum.in.www1.artemis.repository.ModelAssessmentConflictRepository;
+import de.tum.in.www1.artemis.repository.ResultRepository;
+import de.tum.in.www1.artemis.web.rest.errors.EntityNotFoundException;
+
+@Service
+public class ModelAssessmentConflictService {
```

Comment:
Also in this Service I think some more documentation would be good, e.g. what does it mean to escalate a conflict?